refarch-integrations/refarch-email-integration/refarch-email-integration-core/src/main/java/de/muenchen/refarch/email/integration/domain/exception/TemplateError.java

```diff
@@ -1,7 +1,12 @@
 package de.muenchen.refarch.email.integration.domain.exception;
 
-public class TemplateError extends Error {
-    public TemplateError(final String message) {
-        super(message);
+import java.io.Serial;
+
+public class TemplateError extends RuntimeException {
+    @Serial
+    private static final long serialVersionUID = 1L;
```

Comment:
We had a simiilar issue in refarch-templates repository. I think it would be better to suppress the warning for missing serial ID here (instead of adding it) as we are not (de)serializing Exception classes.

Reply:
Done